<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1591091903720' stop='1591092017151'>
    <name>Connection</name>
    <title>Connection</title>
    <test-cases>
        <test-case start='1591091903721' status='passed' stop='1591091948622'>
            <name>In-place with Free plan</name>
            <title>In-place with Free plan</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1591091915982' status='passed' stop='1591091946107'>
                    <name>Can start in-place connection</name>
                    <title>Can start in-place connection</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start='1591091946107' status='passed' stop='1591091948202'>
                    <name>Can assert that site is connected</name>
                    <title>Can assert that site is connected</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
        </test-case>
        <test-case start='1591091948622' status='failed' stop='1591092010666'>
            <name>In-place upgrading a plan from personal to premium</name>
            <title>In-place upgrading a plan from personal to premium</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1591091963883' status='passed' stop='1591091965505'>
                    <name>Can set a sandbox cookie</name>
                    <title>Can set a sandbox cookie</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start='1591091965505' status='passed' stop='1591091994570'>
                    <name>Can start in-place connection</name>
                    <title>Can start in-place connection</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start='1591091994570' status='undefined'>
                    <name>Can process payment for Personal plan</name>
                    <title>Can process payment for Personal plan</title>
                    <attachments>
                        <attachment title='Test failed: Connection :: In-place upgrading a plan from personal to premium' source='8d8c125e-1086-4408-9fe5-216fe6a91cd3-attachment.png' type='image/png' size='7755'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Error: Protocol error (DOM.describeNode): Cannot find context with specified id</message>
                <stack-trace>Error: Protocol error (DOM.describeNode): Cannot find context with specified id
                    at Promise (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/Connection.js:183:56)
                    at new Promise (&lt;anonymous>)
                    at CDPSession.send (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/Connection.js:182:12)
                    at ExecutionContext._adoptElementHandle (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/ExecutionContext.js:204:41)
                    at Frame.waitForSelector (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/FrameManager.js:631:47)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                  -- ASYNC --
                    at Frame.&lt;anonymous> (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/helper.js:111:15)
                    at Page.waitForSelector (/home/travis/build/Automattic/jetpack/node_modules/puppeteer/lib/Page.js:1122:29)
                    at waitForSelector (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:34:30)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
                    at new Promise (&lt;anonymous>)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>